koku/api/settings/utils.py

```diff
@@ -54,21 +54,34 @@ def create_subform(name, title, fields):
     return subform
 
 
-def create_plain_text(name, label):
+def create_plain_text_with_doc(name, label, doc_link):
     """
     Create a plain text field for the settings.
 
     Args:
         (String) name - unique name for switch.
         (String) label - Display text.
+        (Dict) doc_link - documentation props.
 
     Returns:
         [Dict] - plain text component.
     """
-    plain_text = {"component": "plain-text", "name": name, "label": label}
+    plain_text = {"component": "plain-text-with-links", "text": label, "linkProps": [doc_link], "name": name}
     return plain_text
 
 
+def generate_doc_link(path):
+    """
+    Generate cost management link to a given path.
+
+    Args:
+        (String) path - path to the documentation.
+    """
+    version = "4.5"
```

Comment:
Recommendation here: if we have to hard code this consider adding a constant to settings.py and importing that in here to reference the version. That way we can modify it at the top level and have it update anywhere we reference the constant.

Reply:
Thanks! I moved it to `koku.settings` as you suggested.